v1.0/gbfs.json

```
@@ -36,7 +36,20 @@
                   }
                 },
                 "required": ["name", "url"]
-              }
+              },
+              "minItems": 1,
+              "allOf": [
+                {
+                  "type": "array",
+                  "contains": {
+                    "type": "object",
+                    "properties": {
+                      "name": { "const": "system_information" }
+                    },
+                    "required": ["name"]
```

@testower What is the purpose of `"required": ["name"]` here? 

Isn't this already covered by line 38 `"required": ["name", "url"]`?

```suggestion
                    }
```

---

Probably! I'll test without it and report back